packages/manager/apps/pci-private-network/src/api/hooks/useNetwork.ts

```diff
@@ -1,15 +1,20 @@
 import { applyFilters, Filter } from '@ovh-ux/manager-core-api';
 import { PaginationState } from '@ovhcloud/manager-components';
-import { useQuery } from '@tanstack/react-query';
+import { useMutation, useQuery } from '@tanstack/react-query';
 import { useMemo } from 'react';
-import { getAggregatedNetwork, TAggregatedNetwork } from '@/api/data/network';
+import {
+  deleteNetwork,
+  getAggregatedNetwork,
+  TAggregatedNetwork,
+} from '@/api/data/network';
 import { TGateway, TRegion } from '@/api/data/project';
 import { getSubnets, TSubnet } from '@/api/data/subnets';
 import {
   getLocalZoneRegions,
   isLocalZoneRegion,
   paginateResults,
 } from '@/api/utils/utils';
+import queryClient from '@/queryClient';
```

Comment:
Don't import queryClient, use the `useQueryClient` hook
You will create a new instance, not use the one in the context

Reply:
this one is imported from a file defining one instance client 

```
import { QueryClient } from '@tanstack/react-query';

export const queryClient = new QueryClient({
  defaultOptions: {
    queries: {
      staleTime: 300_000,
    },
  },
});

export default queryClient;
```

Reply:
Ok but still not a good practice
Take it from the context, there is a hook to do that